src/styles/sidebar/components/Buttons.scss

```diff
@@ -0,0 +1,68 @@
+@use "sass:map";
+@use "../../variables" as var;
+@use '../../shared/components/Buttons' as buttons;
+
+// An icon-only button with tighter padding (e.g. TopBar icons)
+.CompactIconButton {
+  // Override larger touch-target min size
+  @include buttons.configure(
+    $settings: (
+      'touch-target-size': auto,
+    )
+  );
+  @include buttons.IconButton {
```

Comment:
Here there are additional CSS rules within the button base class, as an additional way to customize button styling (vs. configuration of settings).